src/openrct2/ride/VehicleEntry.h

```diff
@@ -92,9 +93,9 @@ enum : uint32_t
  */
 struct rct_ride_entry_vehicle
 {
-    uint16_t rotation_frame_mask;
-    uint8_t NumRotationFrames;
-    uint8_t num_horizontal_frames; // Appears to be unused, except as a temporary variable in RCT2 (not needed for OpenRCT2)
+    uint16_t TabRotationMask;
+    uint8_t SpriteYawPrecision;
+    uint8_t pad_03;
```

Comment:
We don't really need the padding anymore. Unless you are just keeping this for the next refactor I'd delete it.

Reply:
It is removed in a future PR